### PR TITLE
Polish translation update 

### PR DESCRIPTION
**Pull request by radek-daniluk**
**Summary**
1. Add missing translation strings for polish language.
2. Adjust file formatting to be consistent with english version.
3. Add 'rich text' translation: I think: 'Tekst sformatowany' is more appropriate than ''Edytor wizualny'.
4. Some minor translate fixes

**Test plan**
Maybe @pniedzwiedzinski or someone else can review it.

**A picture of a cute animal (not mandatory but encouraged)**

──────▄▀▄─────▄▀▄
─────▄█░░▀▀▀▀▀░░█▄
─▄▄──█░░░░░░░░░░░█──▄▄
█▄▄█─█░░▀░░┬░░▀░░█─█▄▄█

### CODE REVIEW

File: packages/netlify-cms-locales/src/pl/index.js [hunk 1]
```diff
@@ -162,11 +180,16 @@ const pl = {
     },
   },
   ui: {
+    default: {
+      goBackToSite: 'Wróć dostrony',
```
Comment (pniedzwiedzinski) [inline packages/netlify-cms-locales/src/pl/index.js:184]:
```suggestion
      goBackToSite: 'Wróć do strony',
```

File: packages/netlify-cms-locales/src/pl/index.js [hunk 2]
```diff
@@ -2,9 +2,15 @@ const pl = {
...
     loggingIn: 'Logowanie...',
+    loginWithNetlifyIdentity: 'Zaloguj przez konto Netlify',
+    loginWithBitbucket: 'Zaloguj przez Bitbucket',
+    loginWithGitHub: 'Zaloguj przez GitHub',
+    loginWithGitLab: 'Zaloguj przez GitLab',
     errors: {
       email: 'Wprowadź swój adres email',
       password: 'Wprowadź swoje hasło',
+      identitySettings:
+        'Brak dostępu do ustawień tożsamości. Jeśli używasza backend'u git-gateway upewnij się, że usługa tożsamośći (Identity service) oraz Git Gateway są włączone.',
```
Comment (tomrutgers) [inline packages/netlify-cms-locales/src/pl/index.js:13]: There's an issue with the quotes on this line. Can you have a look and run `yarn format` to resolve other formatting errors? Thanks!

Comment (radek-daniluk) [reply packages/netlify-cms-locales/src/pl/index.js]: Thanks, `yarn format` found one more error. Now should be correct.

**Comment by tomrutgers** [comment]
@erezrokah Not sure why the checks are failing. The formatting is fixed, and the tests and build finish just fine locally..

**Comment by erezrokah** [comment]
> The formatting is fixed, and the tests and build finish just fine locally..

I ran the formatting on my machine and it updated the file, maybe try synced with `master`, then `yarn install` then `yarn format`?

**Comment by erezrokah** [comment]
Replaced by https://github.com/netlify/netlify-cms/pull/3530

**Pull request closed without merging**